src/Microsoft.SqlTools.ServiceLayer/Scripting/ScriptingScriptOperation.cs

```diff
@@ -280,23 +305,24 @@ private void OnPublishModelScriptError(object sender, ScriptEventArgs e)
             Logger.Write(
                 LogLevel.Verbose,
                 string.Format(
-                    "Sending scripting error progress event, Urn={0}, OperationId={1}, Completed={2}, Error={3}",
+                    "Sending scripting error progress event, Urn={0}, OperationId={1}, Sequence={2}, Completed={3}, Error={4}",
                     e.Urn,
                     this.OperationId,
+                    this.eventSequenceNumber,
                     e.Completed,
-                    e.Error));
+                    e?.Error?.ToString()));
```

Comment:
?. syntax returns null if the object is null, right?  Does passing nulls to string.Format work as expected?